tools/cpp/osx_cc_configure.bzl

```diff
@@ -108,6 +109,34 @@ def configure_osx_toolchain(repository_ctx, overriden_tools):
             paths["@bazel_tools//tools/osx/crosstool:cc_toolchain_config.bzl"],
             "cc_toolchain_config.bzl",
         )
+        libtool_check_unique_src_path = str(repository_ctx.path(
+            paths["@bazel_tools//tools/objc:libtool_check_unique.cc"],
+        ))
+        xcrun_result = repository_ctx.execute([
+            "env",
```

Comment:
pull these into a common copts list or something so they can be shared with wrapped_clang?

Reply:
👍